optimade/models/references.py

```diff
@@ -130,7 +247,7 @@ class ReferenceResource(EntryResource):
 
     """
 
-    type: str = Field(
+    type: str = OptimadeField(
```

Comment:
This needs the `queryable` and `support` fields, right?

Reply:
Fixed this and a couple more